ON-40502 # Use key from formsAppConfig if options doesn't exist (#214)
oneblink/forms-cdn: CHANGELOG.md

### Added

- `strict` type checking

> - `googleMapsApiKey` from forms app config used if google map api key had not been set in options

Reviewer: This might be more appropriate in a `Changed` header as:
```suggestion
- Use `googleMapsApiKey` from forms app config if no key is passed in```
